pkgs/development/libraries/webkitgtk/2.14.nix

```diff
@@ -32,29 +47,54 @@ stdenv.mkDerivation rec {
 
   # see if we can clean this up....
 
-  patches = [ ./finding-harfbuzz-icu.patch ];
+  patches = [ ./finding-harfbuzz-icu.patch ]
+  	++ optionals stdenv.isDarwin [
+    ./PR-152650-2.patch
```

Comment:
I don't think we can fetch them directly from macports if that is what you were suggesting.  I had to modify them slightly.  The macports patches are designed to work with `patch -p0`.  I had to add an extra directory to the paths to make them work with `patches` in nix (https://github.com/NixOS/nixpkgs/pull/24700/commits/90f4361fa0dc783c91b8bf2cdf1aefb204ce4893#diff-7e2407d988a384c067352687b99c845e is the commit with that change).